quesma/v2/core/quesma_apis.go

```diff
@@ -75,6 +75,38 @@ type QuesmaBuilder interface {
 	Stop(ctx context.Context)
 }
 
+type Query struct {
```

Comment:
This is just placeholder. Concrete definition will be taken from `quesma/model/query.go`